package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "rollup -c",
     "start": "rollup -c -w",
-    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js --watch"
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
```

Comment:
@jurajk  mne toto celkom vyhovovalo. Máš pre to nejaký dôvod prečo to tam nedávať? Ako zvyknem si len by som chcel vedieť reasoning. Asi kvôli tej automatizácii čo? :) Možno spraviť ešte jeden script s watchom?

Reply:
mozes aj pouzit `npm run test -- --watch` :)

Reply:
ano, ked som to pustil cez github akciu tak to ostalo watchovat. ano, dalo by sa to zlozito cez env vars, ale zdalo sa mi jednoduchsie pustit si to s flagom ako pise adam.

Reply:
@adammockor ano ano poznám aj `-- --watch` ale verím, že to nepozná každý. Je to too deep :D 

Ale napríklad `npm test` by mohol byť na používanie ručne a na automatizáciu by mohlo byť niečo čo aj tak nikto nikdy ručne nespustí :) `test-ci` alebo také niečo. Rozumiete. Ale keby sme chceli byť korektný tak asi test by mal robiť jeden test a watch by malo robiť aj watch nad testami. 

Ale ako ste povedali. Dnes som si to pustil s flagom a hotovo :) Ale otestoval by som či to vedia juniori použiť s flagom.